docs/features/UsedAssemblyReferences.md

```diff
@@ -5,4 +5,7 @@ The *Used Assembly References* feature provides a ```GetUsedAssemblyReferences``
 of metadata assembly references that are considered to be used by the compilation. For example, if a type declared in a
 referenced assembly is referenced in source code within the compilation, the reference is considered to be used. Etc.
 
+Note that documentation must be processed for best results. Since types in xml docs rely on usings, all usings are
+assumed to be used when the usage analysis wasn't performed on doc comments.
```

Comment:
Probably regardless of usings, references in XML doc comments are not taken into account as well?

Reply:
Added. Those false-negatives (misses) are more intuitive, but worth mentioning too.
Overall, this makes me wonder whether this API should be offered at all when documentation isn't turned on.

Reply:
Why not. The references are not used. Removing them doesn't break the build, that is the criteria for "used"

Reply:
The answer is given for specific build options. For example, conditional compilation also affects result. We are not giving the answer for some imaginary situation.

Reply:
I think we should not be talking in terms of "best result", but rather in terms of "what to expect" because including information from doc comments might be not what the consumer actually wants/needs.

Reply:
I suggest the following wording:
>References unique to xml docs are not included when compilation is configured to not process the documentation. Also, all references in usings are assumed to be used under these conditions.

Reply:
> But as the bug report shows, this causes a bit of usability problems especially for false-positives.

I fully support improving the documentation